engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -623,4 +624,11 @@ object WorkflowExecutionActor {
     Props(WorkflowExecutionActor(workflowDescriptor, serviceRegistryActor, jobStoreActor, subWorkflowStoreActor,
       callCacheReadActor, jobTokenDispenserActor, backendSingletonCollection, initializationData, restarting)).withDispatcher(EngineDispatcher)
   }
-}
\ No newline at end of file
+
+  implicit class EnhancedWorkflowOutputs(val outputs: Map[LocallyQualifiedName, WdlValue]) extends AnyVal {
+    def stripLarge = outputs map { case (k, v) =>
+      if (v.toWdlString.length > 1024) (k, WdlString("Output too long, elided"))
```

Comment:
i don't really care what the string says so long as it isn't gigabytes long though, was 'censored' what you really want it to say?